test/README.md

```diff
@@ -1,36 +1,27 @@
 # Testing the code
 
-Test data: ```test/data/small_input_data.csv```
+Test data: `test/data/small_input_data.csv`
 
 - Has 25 samples.
-- All the OTUs that is in the original dataset and 1 column for the outcome (dx).
-- Should take 60-90 seconds to run.
+- All OTUs in the original dataset and 1 column for the outcome (e.g. `dx`).
+- Should take 60-150 seconds to run.
 
 ### To run the test code:
 
+from the command line:
 ```
-Rscript code/R/main.R test/data/test_data.csv 1 "L2_Logistic_Regression" 0 "dx"
+Rscript code/R/main.R --seed 1 --model L2_Logistic_Regression --data  test/data/small_input_data.csv --hyperparams test/data/hyperparams.csv --outcome dx
 ```
```

Comment:
Can you please add a line here to explain how would we run permutation importance if we wanted to?